### PR TITLE
Make blacklist a map for easier membership checking

### PR DESCRIPTION
**Pull request by Zapadlo**

### CODE REVIEW

**Comment by Nesze** [comment]
+1

File: deployer.go [hunk 1]
```diff
@@ -172,7 +172,7 @@ func (d *deployer) destroyUnwanted(wantedUnits map[string]*schema.Unit) error {
 	for _, u := range currentUnits {
 		if wantedUnits[u.Name] == nil {
 			//Do not destroy the deployer itself
-			if !blacklistedService(u.Name, destroyServiceBlacklist) {
```
Comment (martingartonft) [inline deployer.go]: can blacklistedService be removed now?

File: deployer.go [hunk 2]
```diff
@@ -23,7 +23,7 @@ var (
 	fleetEndpoint             = flag.String("fleetEndpoint", "", "Fleet API http endpoint: `http://host:port`")
 	servicesDefinitionFileUri = flag.String("servicesDefinitionFileUri", "", "URI file that contains services definition: `https://raw.githubusercontent.com/Financial-Times/fleet/master/services.yaml`")
 	socksProxy                = flag.String("socksProxy", "", "address of socks proxy, e.g., 127.0.0.1:9050")
-	destroyServiceBlacklist   = []string{"deployer.service", "deployer.timer"}
+	destroyServiceBlacklist   = map[string]bool{"deployer.service": true, "deployer.timer": true}
```
Comment (martingartonft) [inline deployer.go]: how about map[string]struct{}  ?

**Comment by martingartonft** [comment]
lgtm

**Pull request merged**